Dockerfile.thr

```
@@ -0,0 +1,22 @@
+# if node version is changed, also adapt .nvmrc file
+FROM node:8.16-alpine
+
+RUN apk update && apk upgrade && apk add --no-cache autoconf automake build-base git libtool make nasm pkgconfig tzdata zlib-dev
+
+EXPOSE 3100
+
+WORKDIR /home/node/app
+
+COPY ./package.json .
```

```suggestion
COPY ./package.json .
COPY ./package-lock.json .
```
like in the other Dockerfiles